lib/tshield/controllers/requests.rb

```diff
@@ -13,7 +13,7 @@ module TShield
   module Controllers
     # Requests Handler
     module Requests
-      PATHP = %r{([a-zA-Z0-9/\._-]+)}.freeze
+      PATHP = %r{([a-zA-Z0-9/._-]+)}.freeze
```

Comment:
@jerielverissimo could you double check if there is any automated test covering this change?